src/stream/src/executor/error.rs

```diff
@@ -93,6 +89,10 @@ impl StreamExecutorError {
         StreamExecutorErrorInner::InvalidArgument(error.into()).into()
     }
 
+    pub fn connector_error(error: impl Into<String>) -> Self {
```

Comment:
... so that this can also be `Error`.
```suggestion
    pub fn connector_error(error: impl Error) -> Self {
```